Microsoft.Toolkit.Uwp.UI.Controls/InfiniteCanvas/InfiniteCanvas.cs

```diff
@@ -348,7 +349,28 @@ public void ImportFromJson(string json)
         }
 
         /// <summary>
-        /// This event triggered after each render happended because of any change in the canvas elements.
+        /// This method exports the max possible view of the InfiniteCanvas drawings as offScreen drawings that can be converted to image.
+        /// Max is calculated using CanvasDevice.MaximumBitmapSizeInPixels
+        /// </summary>
+        /// <param name="dpi">Dpi that is used in the export process</param>
+        /// <returns>Returns offScreen drawings as <see cref="CanvasRenderTarget"/> that can get converted to image</returns>
+        public CanvasRenderTarget ExportMaxOffScreenDrawings(float dpi = 96)
```

Comment:
1) We shouldn't expose Win2D apis as part of this API.  We should have a single method called `RenderToStream(float dpi = screendpi, enum format = png, bool currentViewOnly = false)` and have them specify the options here.  Then they can choose to save the stream off to a file or some other storage.

Reply:
Instead of an enum, how about `Guid encoderId`